packages/foreman/nodejs-node-gyp/node-gyp-node-dirs.patch

```diff
@@ -0,0 +1,46 @@
+diff --git a/lib/configure.js b/lib/configure.js
+index 2ff476d..7a69e1c 100644
+--- a/lib/configure.js
++++ b/lib/configure.js
+@@ -62,7 +62,9 @@ function configure (gyp, argv, callback) {
+         log.verbose('get node dir', 'compiling against --target node version: %s', release.version)
+       } else {
+         // if no --target was specified then use the current host node version
```

Comment:
should this line be removed/replaced by the next one?

Reply:
I think the two lines are saying two different things if you look at the log messages that go with them.

Reply:
they are, but how can it have two behaviors when no `--target` was passed? or am I reading the diff wrong?

Reply:
Reading this, it seems to be logging multiple assumptions that will happen if "--target" is specified.  My vote is to keep as is